config/initializers/geared_pagination.rb

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+GearedPagination::Engine.config.after_initialize do
+  ActiveSupport.on_load(:action_controller) do
+    ActionController::Base.skip_after_action(:set_paginated_headers)
```

Comment:
Given this kind of implementation detail, I wonder if we're not better off implementing it ourselves.